libs/api/domains/national-registry-x-road/src/lib/computeCountryResidence.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { computeCountryResidence } from './computeCountryResidence'
+
+jest
+  .useFakeTimers('modern')
+  .setSystemTime(new Date('2021-09-18T00:00:00').getTime())
+
+const d = new Date().getTime()
+
+// helper to return a date object with subtracted days from "today"
+const makeDate = (addDays: number): Date => new Date(d + addDays * 86400 * 1000)
```

Comment:
Nitpick, but would have the constants 86400 and 1000 as varibles for readability

Reply:
I think you could also do it like this: 

```
const d = new Date().getDate()

const makeDate = (addDays: number): Date => new Date(d + addDays)
```

Reply:
I think that wouldn't work, but I might be misunderstanding :)

But I'll add constants, thanks